support-lib/jni/djinni_support.cpp

```diff
@@ -50,6 +50,13 @@ JNIEnv * jniGetThreadEnv() {
     assert(g_cachedJVM);
     JNIEnv * env = nullptr;
     const jint get_res = g_cachedJVM->GetEnv(reinterpret_cast<void**>(&env), JNI_VERSION_1_6);
+
+    // PSPDFKit change - Attach thread if it's not attached so we can do callbacks from libdispatch.
```

Comment:
I've talked to guys and due to potential issues in general use-case they're not really considering merger of this.